test/nonregression/testing_tools.py

```diff
@@ -111,20 +111,59 @@ def similarity_measure(
     from os import fspath
 
     import nibabel
-    from skimage.metrics import structural_similarity
 
     image1 = nibabel.load(fspath(file1)).get_fdata()
     image2 = nibabel.load(fspath(file2)).get_fdata()
-    data_range = image1.max() - image1.min()
 
-    # See https://scikit-image.org/docs/stable/api/skimage.metrics.html#skimage.metrics.structural_similarity
+    return similarity_measure_arrays(image1, image2, threshold)
+
+
+def similarity_measure_large_nifti(
+    file1: PathLike,
+    file2: PathLike,
+    threshold: float,
+) -> bool:
+    """TODO"""
```

Comment:
Let's not forget this !

Reply:
Done in [66edcd6](https://github.com/aramis-lab/clinica/pull/902/commits/66edcd680651598c1da64834658c66aac8905349)